home-open-api/src/main/resources/yaml/openapi.yaml

```diff
@@ -1412,6 +1414,175 @@ paths:
         default:
           $ref: '#/components/responses/InternalServerError'
+  /cooperation:
+    post:
+      tags:
+        - cooperation
+      summary: Create cooperation
+      description: Create a new cooperation
+      operationId: createCooperation
+      requestBody:
+        description: Create a new cooperation
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/CreateCooperation'
+        required: true
+      responses:
+        '201':
+          description: Successful created
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/ReadCooperation'
+        '400':
+          $ref: '#/components/responses/BadRequest'
+        "401":
+          $ref: '#/components/responses/Unauthorized'
+        "403":
+          $ref: '#/components/responses/Forbidden'
+        "409":
+          $ref: '#/components/responses/Conflict'
+        default:
+          $ref: '#/components/responses/InternalServerError'
+    get:
+      tags:
+        - cooperation
+      summary: "Get all cooperation"
+      description: ''
+      operationId: queryCooperation
+      parameters:
+        - in: query
+          name: pageNumber
+          schema:
+            type: integer
+            minimum: 1
+            default: 1
+          required: false
+          description: The number of current page.
+        - in: query
+          name: pageSize
+          schema:
+            type: integer
+            minimum: 0
+            maximum: 10
+            default: 5
+          required: false
+          description: The number of items to return.
+      responses:
+        '200':
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/ReadCooperation"
+        '400':
+          $ref: '#/components/responses/BadRequest'
+        "401":
+          $ref: '#/components/responses/Unauthorized'
+        default:
+          $ref: '#/components/responses/InternalServerError'
+  /cooperation/{cooperation_id}:
+    get:
+      tags:
+        - cooperation
+      summary: "Get cooperation by cooperation name"
+      description: ''
+      operationId: getCooperation
+      parameters:
+        - name: "cooperation_id"
+          in: "path"
+          description: "Use 1 for testing."
+          required: true
+          schema:
+            type: integer
+            format: int64
+            example: 1
+      responses:
+        '200':
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ReadCooperation"
+        '400':
+          $ref: '#/components/responses/BadRequest'
+        "401":
```

Comment:
If we following the approach to documenting most of the possible errors, can forbidden appear here?